src/packages/code-editor/property-editor/manifests.ts

```diff
@@ -21,14 +21,14 @@ export const manifest: ManifestPropertyEditorUi = {
 						{
 							alias: 'items',
 							value: [
-								{ name: 'JavaScript', value: 'javascript', selected: true },
-								{ name: 'HTML', value: 'html' },
-								{ name: 'TypeScript', value: 'typescript' },
 								{ name: 'C#', value: 'csharp' },
-								{ name: 'Razor', value: 'csharp' },
 								{ name: 'CSS', value: 'css' },
-								{ name: 'Markdown', value: 'markdown' },
+								{ name: 'HTML', value: 'html' },
+								{ name: 'JavaScript', value: 'javascript' },
 								{ name: 'JSON', value: 'json' },
+								{ name: 'Markdown', value: 'markdown' },
+								{ name: 'Razor (CSHTML)', value: 'razor' },
```

Comment:
@leekelleher I haven't used `Blazor`, but couldn't it possible be used on `.razor` (client side) as well if using code editor for custom stuff?
https://stackoverflow.com/a/67219348

Reply:
Yes, .razor should work as well